Makefile

```diff
@@ -612,9 +612,11 @@ run.skaffold:
 		SKAFFOLD_PROFILE=$(SKAFFOLD_RUN_PROFILE) \
 		$(MAKE) _skaffold
 
+# NOTE: We're using the --keep-running-on-failure=true to allow deployments like
```

Comment:
Can we please add a reference to https://github.com/Kong/kubernetes-ingress-controller/issues/5116 here with some text saying to revisit whether `--keep-running-on-failure=true` is still needed once #5116 is implemented?

Reply:
👍 Added. PTAL